pkg/clusteragent/clusterchecks/dispatcher_rebalance.go

```diff
@@ -32,6 +32,17 @@ func (w Weights) Len() int           { return len(w) }
 func (w Weights) Less(i, j int) bool { return w[i].busyness > w[j].busyness }
 func (w Weights) Swap(i, j int)      { w[i], w[j] = w[j], w[i] }
 
+type RebalancingDecision struct {
```

Comment:
it has the same fields as `RebalanceResponse`, could we just keep one with the json fields ?